python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -275,6 +276,33 @@ class PipesSession:
     message_reader_params: PipesParams
     context: OpExecutionContext
 
+    @property
+    def dagster_tags(self) -> Dict[str, str]:
+        """Important Dagster key-value pairs typically attached to external resources launched from Pipes."""
+        tags = {
+            "dagster.io/run_id": self.context.run_id,
+            "dagster.io/job_name": self.context.job_name,
+        }
+
+        if user := self.context.get_tag("dagster/user"):
+            tags["dagster.io/user"] = user
+
+        if self.context.has_partition_key:
```

Comment:
We generally prefix with `dagster/` rather than `dagster.io`. We should be consistent with the core framework.

Also what is the use case for this?

Reply:
I's a good practice to provide this information to users. It makes various tasks like costs tracking, resource cleanup, etc, easier. 

It's very common in Kubernetes development. 

For example, I was personally  Kubernetes labels to identify and cleanup old temporary resources created by Dagster which were not properly destroyed for some reason.